scripts/pin_the_slow_way.py

```diff
@@ -41,7 +41,7 @@
           'glib': 'glib 2.51.*',  # 2.50.2
           'gmp': 'gmp >=5.0.1,<7', # 6.1.0
           'harfbuzz': 'harfbuzz 1.3.*',  # 0.9.39
-          'hdf5': 'hdf5 1.8.17|1.8.17.*',  # 1.8.17
+          'hdf5': 'hdf5 1.8.18|1.8.18.*',  # 1.8.17
```

Comment:
It is still 1.8.17 - I have a PR to update it here: https://github.com/conda-forge/hdf5-feedstock/pull/68

In my experience, my PRs get ignored even if it's about a remote code execution vuln.